assets/app/views/settings.html

```diff
@@ -110,8 +141,6 @@ <h3 class="text-center">Memory</h3>
                             }">
                           <td>
                             <span ng-switch="resourceType" class="hide-ng-leave">
```

Comment:
seems like this shouldn't be an ng-switch anymore if the only thing in it is ng-switch-default?